scripts/run_e2e_tests.py

```diff
@@ -430,6 +441,20 @@ def start_google_app_engine_server(dev_mode_setting):
     SUBPROCESSES.append(p)
 
 
+def get_chrome_driver_version():
+    """Fetches the latest supported version of chromedriver depending on the
+    Chrome version.
+    https://chromedriver.chromium.org/downloads/version-selection
```

Comment:
Could you also mention what this links to, e.g. that this function uses the interface described at the link?

Reply:
Done.